core/src/main/java/org/apache/calcite/adapter/enumerable/RexImpTable.java

```diff
@@ -3061,6 +3063,28 @@ private static class ReinterpretImplementor extends AbstractRexCallImplementor {
     }
   }
 
+  /** Implementor for sort_array. */
+  private static class SortArrayImplementor extends AbstractRexCallImplementor {
+    SortArrayImplementor() {
+      super("sort_array", NullPolicy.STRICT, false);
+    }
+
+    @Override Expression implementSafe(RexToLixTranslator translator,
+        RexCall call, List<Expression> argValueList) {
+      if (argValueList.size() == 2) {
+        return Expressions.call(
+            BuiltInMethod.SORT_ARRAY.method,
+            argValueList.get(0),
```

Comment:
Pretty sure if you're just passing the whole argValueList you don't need to retrieve each element individually, you can just do `.call(BuiltInMethod.SORT_ARRAY.method, argValueList);` The `LogicalNotImplementor` and `LogImplementor` are a couple of examples.

Reply:
https://github.com/apache/calcite/pull/3236/files#diff-23301236e7a9c0e772620a2093cce6a979d7703c511f2417973fae0b26ee62b3R3891

```
list: argValueList.get(0)
ordering: argValueList.get(1) 
```

Reply:
Here is an example: you call the method and pass just the entire `argValueList` https://github.com/apache/calcite/blob/main/core/src/main/java/org/apache/calcite/adapter/enumerable/RexImpTable.java#L3884

The function takes two arguments: https://github.com/apache/calcite/blob/main/core/src/main/java/org/apache/calcite/util/BuiltInMethod.java#L449

But the argValueList has two elements so this matches.